ifrs17/Import/Importers.ipynb

```diff
@@ -596,7 +631,10 @@
                 "\n                                                                                        FunctionalCurrency = storage.ReportingNode.Currency,",
                 "\n                                                                                        LineOfBusiness = datarow.Field<string>(nameof(DataNode.LineOfBusiness)),",
                 "\n                                                                                        ValuationApproach = datarow.Field<string>(nameof(DataNode.ValuationApproach)),",
-                "\n                                                                                        OciType = datarow.Field<string>(nameof(DataNode.OciType))",
+                "\n                                                                                        OciType = datarow.Field<string>(nameof(DataNode.OciType)),",
+                "\n                                                                                        Scenario = scenarioColumnInsurancePortfolio",
```

Comment:
Is this the dimension? Scenario should not be in a dimension, only parameters and state.

Reply:
Don't we want to support having scenario GOC?

Reply:
This should only affect the parameter (data node parameter). You can add a GIC in the dimension and control in the state if it should be active or not. The dimension should go across all scenarios, as to me this is the definition (same as for any other partition, like year, month, etc. --> if you change a Lob of a GIC, it will have an impact in all reporting periods. 

Please, make the dimension scn independent, and only allow scn for parameters.

Reply:
done